l10n_account_translate/README.rst

```diff
@@ -0,0 +1,7 @@
+Description
+===========
+
+This module sets translate=True on the account.account,name field
```

Comment:
using syntax like 'account.account.name' doesn't make a clear difference between the model and the field name. That's why I used account.account,name (same syntax as used technically in the ir.translation table).
Since it seems confusing I changed the description as follows:
This module sets translate=True on the General Account (account.account) name field